src/Symfony/Bundle/FrameworkBundle/Resources/config/cache_pools.xml

```diff
@@ -10,25 +10,6 @@
             <tag name="kernel.cache_clearer" />
         </service>
 
-        <service id="cache.adapter.shared" alias="cache.adapter.filesystem" />
-        <service id="cache.adapter.local" alias="cache.adapter.filesystem" />
-
-        <service id="cache.pool.shared" parent="cache.adapter.shared">
-            <tag name="cache.pool" clearer="cache.default_pools_clearer" />
-        </service>
-
-        <service id="cache.pool.local" parent="cache.adapter.local">
-            <tag name="cache.pool" clearer="cache.default_pools_clearer" />
-        </service>
-
-        <service id="cache.pool.validator" parent="cache.adapter.local" public="false">
-            <tag name="cache.pool" clearer="cache.default_pools_clearer" />
-        </service>
-
-        <service id="cache.pool.serializer" parent="cache.adapter.local" public="false">
-            <tag name="cache.pool" clearer="cache.default_pools_clearer" />
-        </service>
-
         <service id="cache.adapter.apcu" class="Symfony\Component\Cache\Adapter\ApcuAdapter" abstract="true">
```

Comment:
Let's at the cache.pool tag to base adapters, that should help e.g. the phpstorm plugin provide autocompletion